packages/chat-client/src/utils/jwtAuth.ts

```diff
@@ -0,0 +1,146 @@
+import { base58btc } from "multiformats/bases/base58";
+import bs58 from "bs58";
+import * as ed25519 from "@noble/ed25519";
+import { isValidObject } from "@walletconnect/utils";
+
+interface JwtHeader {
```

Comment:
Is there a reason for not using the [did-jwt library](https://www.npmjs.com/package/did-jwt) ?

Reply:
Its last publish was 2 months ago, has <300 stars, only has 17k weekly downloads and this felt more sustainable if we were to ever change anything in the future.

Reply:
TIL that 17k weekly downloads is weak sauce! 😂